packages/engine/src/interaction/components/InteractableComponent.ts

```diff
@@ -60,6 +68,33 @@ export enum XRUIActivationType {
   proximity = 0,
   hover = 1
 }
+
+/**
+ * Adds an interactable UI to the entity if it has label text
+ * @param entity
+ */
+const addInteractableUI = (entity: Entity) => {
+  if (!isClient || getMutableState(EngineState).isEditing.value) return //no xrui in editor
+  const interactable = getComponent(entity, InteractableComponent)
+  if (!interactable.label || interactable.label === '' || interactable.uiEntity != UndefinedEntity) return //null or empty label = no ui
+
+  interactable.uiEntity = createUI(entity, interactable.label, interactable.uiInteractable).entity
+  setComponent(interactable.uiEntity, EntityTreeComponent, { parentEntity: Engine.instance.originEntity })
+
+  const transition = createTransitionState(0.25)
+  transition.setState('OUT')
+  InteractableTransitions.set(entity, transition)
+}
+
+const removeInteractableUI = (entity: Entity) => {
+  if (!isClient || !getMutableState(EngineState).isEditing.value) return //no xrui in editor
```

Comment:
unnecessary checks

Reply:
removing the !isClient || !isEditing check.